models/Recipe.model.js

```diff
@@ -0,0 +1,82 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const recipeSchema = new Schema(
+  {
+    title: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+
+    level: {
+      type: String,
+      enum: ['Easy Peasy', 'Amateur Chef', 'UltraPro Chef'],
+      required: true,
+    },
+
+    ingredients: {
+      type: Array,
+      required: true,
+    },
+    directions: {
+      type: Array,
+      required: true,
+    },
+    ratings: [
+      {
+        type: Number,
+        min: 0,
+        max: 5,
+      },
+    ], // [{user1ID: number}, {user2ID: number}, ...]
+
+    ratingAverage: Number,
```

Comment:
ratingAverage I wouldn't suggest to store into the database, instead you can compute it every time from the list of ratings when you display the recipe